src/payment/create-payment-strategy-registry.ts

```diff
@@ -431,7 +432,9 @@ export default function createPaymentStrategyRegistry(
             createGooglePayPaymentProcessor(
                 store,
                 new GooglePayCheckoutcomInitializer(requestSender)
-            )
+            ),
+            undefined,
```

Comment:
Instead of adding arguments, can we instead implement a common interface that all payment processors can implement?